lerna.json

```diff
@@ -1,8 +1,4 @@
 {
-  "packages": [
-    "configs/*",
-    "packages/*",
-    "scripts/docker"
-  ],
-  "version": "0.0.0-dev.0"
+  "packages": ["configs/*", "packages/*", "scripts/docker"],
+  "version": "independent"
```

Comment:
I'm a bit unsure how to go abut all this; 
I think this may and maybe a couple more changes might be all we need.

But I tried i.e. running `npm run bump-packages` locally and I wonder if it is because of the fact that this PR itself technically changes all of the packages, we end up with a bunch of bumps; some reasonable others not so much.

I assume some of these packages we'd like to ignore right? And also some which we should keep at `0.0.0-dev.0` for our development?

Reply:
> I assume some of these packages we'd like to ignore right? And also some which we should keep at 0.0.0-dev.0 for our development?

Well, the `mongosh` and `@mongosh/cli-repl` packages specifically should still be bumped as part of the process we have now, that can be by using the `0.0.0-dev.0` placeholders but I don't think that's a strict requirement here